protected/humhub/modules/stream/models/StreamQuery.php

```diff
@@ -525,6 +526,12 @@ protected function setupCriteria()
             ->joinWith('contentContainer')
             ->limit($this->limit);
 
+        // Filter out content created by not enabled users
+        $this->_query->andWhere(['OR',
+            ['IS', 'user.id', new Expression('NULL')],
+            ['user.status' => User::STATUS_ENABLED],
+        ]);
+
```

Comment:
What about the condition below? Isn't it responsible for such users?

Reply:
@luke- Yes, you are right the below condition is same, but it is not by default, i.e. by default contents of disabled users are visible in stream.
Should we keep it as it was before of change the property `humhub\modules\stream\Module::$showDeactivatedUserContent` to `false` by default?

Reply:
@yurabakhtin I've created another issue for this here: https://github.com/humhub/team_tasks/issues/351 Please keep the flag as it is.

Reply:
@luke- Ok, then I will remove the current code as it is a duplication.